<launch>
  <!-- slam_map to odom tf -->
  <node pkg="tf" type="static_transform_publisher" name="slam_map_to_odom_publisher" args="0 0 0 0 0 0 slam_map odom 20">
  </node>
</launch>
  
